sources/README.md

```diff
@@ -667,6 +667,20 @@ _Service Centre of the Federal Government for Geo-Information and Geodesy_
 * license: _https://www.govdata.de/dl-de/by-2-0_
 * usage: `concordance`, `property`, `geometry`
 
+## Landesamt für Digitalisierung, Breitband und Vermessung
+
+_Bavarian state office for digitisation, broadband and measurement_ 
+
+* added: `2023-03-17`
+* id: `1830479083`
+* name: `de-by`
+* prefix: `de-by`
```

Comment:
I don't have strong opinions here, but we could also consider:

- `de-ldbv`

Where `de` is the country code context, and `ldbv` is the organization abbreviation for "Bavarian State Office for Digitization, Broadband and Surveying" (versus just saying the states abbreviation).

(This would need to be changed a few places...)